tutorials/notebooks/keras/ptq/keras_deeplabv3plus_for_imx500.ipynb

```diff
@@ -0,0 +1,342 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "4c261298-309f-41e8-9338-a5e205f09b05",
+   "metadata": {},
+   "source": [
+    "# DeeplabV3+ Semantic Segmentation Keras Model - Quantization for IMX500\n",
+    "\n",
+    "[Run this tutorial in Google Colab](https://colab.research.google.com/github/sony/model_optimization/blob/main/tutorials/notebooks/keras/ptq/keras_deeplabv3plus_for_imx500.ipynb)\n",
+    "\n",
+    "## Overview\n",
+    "\n",
+    "In this tutorial, we'll quantize the DeeplabV3+ model for semantic segmentation with MCT's post-training quantization techniques. The pretrained model was generated with this [repo](https://github.com/bonlime/keras-deeplab-v3-plus), after making the following changes:\n",
+    "\n",
+    "1. input image size set to 320x320\n",
+    "2. expand_dims in Lambda layer replaced with a simple Reshape layer\n",
+    "3. tf.compat.v1.image.resize in Lambda layer replaced with a Resizing layer\n",
+    "4. Added argmax at model output"
+   ]
```

Comment:
I think it better to give more high level explanation:
1. input image size set to 320x320
2. Incompatible layers were replaced
3. Argmax was added at the model output (why? maybe a few words...)